HBPreferences.m

```diff
@@ -89,7 +82,14 @@ - (instancetype)initWithIdentifier:(NSString *)identifier {
 
 		KnownIdentifiers[_identifier] = self;
 
-		CFNotificationCenterAddObserver(CFNotificationCenterGetDarwinNotifyCenter(), NULL, (CFNotificationCallback)HBPreferencesDarwinNotifyCallback, (CFStringRef)[_identifier stringByAppendingPathComponent:@"ReloadPrefs"], NULL, kNilOptions);
+		int token, status;
```

Comment:
`status` is actually `int32_t`. (See [notify(3)](https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man3/notify.3.html))

However, it doesn’t seem we need to store this. Unless there’s a reason for it, remove the definition and assignment.